src/Tribe/Service_Providers/Tooltip.php

```diff
@@ -16,34 +19,25 @@ class Tooltip extends \tad_DI52_ServiceProvider {
 	 * @since 4.9.8
 	 */
 	public function register() {
-		tribe_singleton( 'tooltip.view', '\Tribe\Tooltip\View' );
+		tribe_singleton( 'tooltip.view', View::class );
 
-		$this->hook();
+		$this->register_assets();
 	}
 
 	/**
-	 * Setup hooks for classes.
+	 * Registers the assets for the Tooltip class.
 	 *
-	 * @since 4.9.8
+	 * @since TBD
 	 */
-	private function hook() {
-		add_action( 'tribe_common_loaded', [ $this, 'add_tooltip_assets' ] );
-	}
-
-	/**
-	 * Register assets associated with tooltip
-	 *
-	 * @since 4.9.8
-	 */
-	public function add_tooltip_assets() {
-		$main = \Tribe__Main::instance();
+	public function register_assets() {
+		$main = Common::instance();
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: f48620d95abe6ea232d155d7eb6aaa84</sub>